client/src/components/Slider/Slider.js

```diff
@@ -0,0 +1,107 @@
+import imageFile1 from "../../Images/Fall-2020-1.png";
+import imageFile2 from "../../Images/Fall-2020-2.png";
+import imageFile3 from "../../Images/Spring-2020-1.jpeg";
+import imageFile4 from "../../Images/Spring-2020-2.jpeg";
+import imageFile5 from "../../Images/Trending-Now-1.jpeg";
+import imageFile6 from "../../Images/Trending-Now-2.jpeg";
+import imageFile7 from "../../Images/Classics-1.jpeg";
+import imageFile8 from "../../Images/Classics-2.jpeg";
+
+import React from "react";
+import ReactDOM from "react-dom";
+import "./Slider.css";
+
+const slides = {
+  properties: [
+    {
+      title: "Fall 2020 Collection",
+      imga: imageFile1,
+      imgb: imageFile2,
+      labela: "LUCA",
+      labelb: "VIDA",
+      index: 0
+    },
+    {
+      title: "Spring 2020 Collection",
+      imga: imageFile3,
+      imgb: imageFile4,
+      labela: "AlWAYS",
+      labelb: "BELIZE",
+      index: 1
+    },
+    {
+      title: "Trending Now",
+      imga: imageFile5,
+      imgb: imageFile6,
+      labela: "ALICE",
+      labelb: "EVELYN",
+      index: 2
+    },
+    {
+      title: "Classics",
+      imga: imageFile7,
+      imgb: imageFile8,
+      labela: "AINSLEY",
+      labelb: "ANNIE",
+      index: 3
+    }
+  ]
+};
```

Comment:
All of this should be passed down as props

Reply:
Same question as with the images, should I make a file for images and the parsed array of data  for the slides and keep that separate from the Slider.js file or pass down props from within Slider.js?

Reply:
This set of data should all be handled in the `Stories.js` file. Not sure why you would need to create another file for this.

Reply:
Reply here is for the import images in `Stories.js` comment above as well as both are essentially the same edit.  I have put in the lines to handle importing and structuring this data in `Stories.js`, but passing as props to `Slider.js` has been a bit tricky so far with declaring and I am still pulling directly in `Slider.js`.  This is the last edit I need to handle before I can submit again for code review (with a couple touch ups when I can make this work) please advise if you have any suggestions for reading or otherwise, I would like to wrap this up as early as possible on Friday so I can submit and get to work on the Molecule I am assigned that uses this this Atom and with any luck start tackling a third Component in the afternoon(I am optimistic).